Rubberduck.Refactorings/ExtractInterface/ExtractInterfaceRefactoring.cs

```diff
@@ -67,7 +68,8 @@ private void AddInterface(ExtractInterfaceModel model)
             var suspendResult = _parseManager.OnSuspendParser(this, new[] {ParserState.Ready}, () => AddInterfaceInternal(model));
             if (suspendResult != SuspensionResult.Completed)
             {
-                _logger.Warn("Extract interface failed.");
+                _logger.Warn($"Extract interface failed because a parser suspension request could not be fulfilled.The request's result was '{suspendResult.ToString()}'.");
+                throw new OperationCanceledException(RubberduckUI.RefactoringFailure_BaseMessage);
```

Comment:
Please note that this will not notify the user. Instead it will be logged again as an error by `CommandBase`.

If you want to notify the user, you need to use a (possibly new) subclass of `RefactoringException`. Preferably, the corresponding notifier would be updated, too. (Messages presented to the user are not a concern of the refactoring itself.)